examples/automm/pipeline/feature_extraction_example.py

```diff
@@ -48,4 +69,9 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--checkpoint_name", default="sentence-transformers/msmarco-MiniLM-L-12-v3", type=str)
+    parser.add_argument("--onnx_path", default="/media/code/autogluon/examples/automm/production/sentence-transformers_msmarco-MiniLM-L-12-v3.onnx", type=str)
```

Comment:
Should we just use `sentence-transformers_msmarco-MiniLM-L-12-v3.onnx` ?

Reply:
Here I'm trying to show two ways of feature extraction: MultiModalPredictor v.s. onnx. checkpoint_name is for MultiModalPredictor. I updated the code to infer a default onnx_path using checkpoint_name.
Also I'd like to update that there is a large inference time gap between onnx and MultiModalPredictor, for this STSBenchmark task the time difference (model=minilm-l-12) is:
Autogluon load time: 3.8375
Autogluon eval time: 16.1105
ONNX load time: 1.2615
ONNX eval time: 2.2466